ex8/phonebook.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="style/style.css">
+  <script data-main="js/config" src="js/lib/require/require.js"></script>
```

Comment:
`data-main` should be a javascript file that actually does the work (the several sines below: the calls for ajax_get, insertJson). Please move these calls into config and rename it to be `main.js`.